pilot/pkg/serviceregistry/kube/controller/pod.go

```diff
@@ -147,6 +148,16 @@ func (pc *PodCache) getPodByIP(addr string) *v1.Pod {
 	return item.(*v1.Pod)
 }
 
+// getPod loads the pod from k8s.
+func (pc *PodCache) getPod(name string, namespace string) *v1.Pod {
+	pod, err := pc.c.client.CoreV1().Pods(namespace).Get(name, metav1.GetOptions{})
+	if err != nil {
+		log.Warnf("Pod %s in namespace %s does not exist in k8s api server", name, namespace)
```

Comment:
how about`failed to get pod  %s/%s from kube-apiserver: %v`

Reply:
ok. change